.github/workflows/autodoc.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: Autodoc
+
+on: [push]
+
+jobs:
+  autodoc:
+    name: Autodoc
+    if: github.repository == 'aibasel/downward'
```

Comment:
Alternatively, we could use if: github.ref == 'refs/heads/main' because we only want to run this job after pushes to the main branch. Or actually, we should use both conditions, to cover the case that someone pushes an issue branch to the aibasel repo, for which we wouldn't want to run autodoc, and to still exclude the whole workflow from being run on forks.

Reply:
I think too that we need both checks (pushing an issue branch can easily happen, and pushing a release branch is intended).

I prefer the branch check to use the ``` on: ...``` syntax, because I think it is easier to read and the more frequently used approach (the on thing is described everywhere, the if: github.X rarely).

Would this resolve your comment or did I miss something?

Reply:
Fine.